src/main/resources/helptexts/contributors.html

```diff
@@ -1,5 +1,26 @@
-<h1>Test header</h1>
-<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit. Pellentesque semper nulla eu placerat consectetur.</p>
+<h1>Contributor</h1>
```

Comment:
We've merged the Creator and Contributor field. We'll need to merge these two explanations as well. For now, can you use this helptext, and add this sentence: `To indicate the person and organization responsible for the dataset choose the role 'creator'.`